tx/textual/valuerenderer/encode.go

```diff
@@ -0,0 +1,36 @@
+package valuerenderer
+
+import (
+	"io"
+
+	"cosmossdk.io/tx/textual/cbor"
+)
+
+var (
+	textKey   = cbor.NewUint(1)
+	indentKey = cbor.NewUint(2)
+	expertKey = cbor.NewUint(3)
+)
+
+// Encode encodes an array of screens according to the CDDL
+func Encode(screens []Screen, w io.Writer) error {
```

Comment:
Same comment as above, in the spirit of keeping the API surface small, I would keep this as private for now.

In my mind, Textual should expose only:
- one `Textual` struct (or maybe interface), with 3 methods `GetSignBytes(Context, Tx) ([]byte, error)`, `DefineScalar` and `DefineMessageRenderer`.
- all the `New*ValueRenderers` 

But we can explore API improvements in #13700

Reply:
Done.